ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/DistributingExecutor.java

```diff
@@ -214,7 +223,8 @@ public StatementExecutorResponse execute(
           injected.getStatementText(),
           queuedCommandStatus.getCommandId(),
           commandStatus,
-          queuedCommandStatus.getCommandSequenceNumber()
+          queuedCommandStatus.getCommandSequenceNumber(),
+          getDeprecatedWarnings(executionContext.getMetaStore(), injected)
```

Comment:
Why do we need to write the warning into the command topic? -- I thought we want to send it back to the client that issue the statement?

Reply:
This is the `StatementExecutorResponse`. It is not written to the command topic but sent back to the client.